src/mame/handheld/hh_ht11xx.cpp

```diff
@@ -42,33 +42,72 @@ class hh_ht11xx_state : public driver_device
 public:
 	hh_ht11xx_state(const machine_config &mconfig, device_type type, const char *tag) :
 		driver_device(mconfig, type, tag),
-		m_maincpu(*this, "maincpu"),
-		m_out_x(*this, "%u.%u", 0U, 0U),
-		m_in(*this, "IN%u", 1)
+		m_maincpu(*this, "maincpu")
 	{ }
```

Comment:
Constructors of abstract classes should be protected (this class has no public machine configuration member functions and hence can’t be used as a state class directly.